tests/components/totalconnect/snapshots/test_alarm_control_panel.ambr

```diff
@@ -41,7 +41,7 @@
       'code_format': None,
       'cover_tampered': False,
       'friendly_name': 'test',
-      'location_id': '123456',
+      'location_id': 123456,
```

Comment:
Seems to change from a string to an int. Is this intended?

Reply:
In previous tests this was set as a `str` and silently caused no errors.  However, with the new code and tests in this PR I discovered it should have been an `int` all along.

Reply:
Epenet opened a PR to change that and I merged that today FYI